flytekit/__init__.py

```diff
@@ -183,6 +183,7 @@
 from flytekit.core.workflow import WorkflowFailurePolicy, reference_workflow, workflow
 from flytekit.deck import Deck
 from flytekit.extras.persistence import GCSPersistence, HttpPersistence, S3Persistence
+from flytekit.extras.pytorch import PyTorchModuleTransformer, PyTorchTensorTransformer
```

Comment:
I think this will break if torch isn't installed (?)

Maybe we can do something like:

```python
from flytekit.extras import pytorch
```

Which would automatically register the tensor and module type transformers (if `torch` is installed)

Reply:
My bad. Fixed the import now.